Tools/CISupport/ews-build/events.py

```diff
@@ -314,9 +315,41 @@ class GitHubEventHandlerNoEdits(GitHubEventHandler):
     ACTIONS_TO_TRIGGER_EWS = ('opened', 'synchronize')
     OPEN_STATES = ('open',)
 
+    @classmethod
+    def file_with_status_sign(cls, info):
+        if info.get('status') in ('added', 'modified'):
+            return '+++ {}'.format(info['filename'])
+        return '--- {}'.format(info['filename'])
+
     def _get_commit_msg(self, repo, sha):
         return ''
 
+    @defer.inlineCallbacks
+    def _get_pr_files(self, repo, number):
+        # Copied from buildbot/www/hooks/github.py to include added/modified/deleted
```

Comment:
Let's have the url here for quick reference, e.g.:  https://github.com/buildbot/buildbot/blob/v2.10.5/master/buildbot/www/hooks/github.py